src/event_handler.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+import json
+import logging
+import os
+
+from base64 import b64decode
+from urllib.request import Request, urlopen
+from urllib.error import URLError, HTTPError
+
+
+# The base-64 encoded, encrypted key (CiphertextBlob) stored in the kmsEncryptedHookUrl environment variable
+ENCRYPTED_HOOK_URL = os.environ['kmsEncryptedHookUrl']
+SLACK_CHANNEL = os.environ['slackChannel']
+
+HOOK_URL = "https://" + boto3.client('kms').decrypt(CiphertextBlob=b64decode(ENCRYPTED_HOOK_URL))['Plaintext'].decode('utf-8')
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def notify_slack(event, context):
```

Comment:
How does this get called? Does it have to be configured in the lambda? (In which case, is that worth mentioning in the readme?)

Reply:
That is defined in event infrastructure. When Robin finishes his PR I'll link to it.